net/ssl/ssl_key_logger_impl.cc

```diff
@@ -50,7 +50,11 @@ class SSLKeyLoggerImpl::Core
   Core& operator=(const Core&) = delete;
 
   void SetFile(base::File file) {
-    // file_.reset(base::FileToFILE(std::move(file), "a"));
+#if defined(STARBOARD)
```

Comment:
Consider using COBALT_PENDING_CLEAN_UP if this is something we potentially would want to implement correctly